cni/87-podman-bridge.conflist

```diff
@@ -4,7 +4,7 @@
     "plugins": [
       {
         "type": "bridge",
-        "bridge": "cni0",
+        "bridge": "cni-podman0",
```

Comment:
Does `/usr/share/conainers/libpod.conf` need to be updated as well?

https://github.com/containers/libpod/blob/master/libpod.conf#L66

Reply:
So long as the `"name": "podman"` portion remains the same, it should be fine